test/mako/alerter/slack/message.go

```diff
@@ -85,10 +86,14 @@ func (smh *MessageHandler) SendAlert(testName, summary string) error {
 			); err != nil {
 				errCh <- fmt.Errorf("failed to retrieve message history in channel %q", channel.Name)
 			}
-			// do not send message again if messages were sent on the same channel a short while ago
-			if len(messageHistory) != 0 {
-				return
+			// do not send message again if alert for this test has been sent to
+			// the channel a short while ago
+			for _, message := range messageHistory {
+				if strings.Contains(message, testName) {
```

Comment:
This doesn't seem like a very reliable way, is there a better way of extracting testName from message?

Reply:
Or we add a function to add brackets around the test name, and also use the result for checking? WDYT?

Reply:
That should work, would be nice to:
- have a unit test guard against brackets from somewhere else
- make sure test full name doesn't contain brackets

Reply:
Done.  Test full name won't contain brackets, and it will still work even if it contains.